source/includes/_dataset_status.md

```diff
@@ -0,0 +1,307 @@
+# Dataset status
+
+## The dataset status object
+
+A dataset status describe the current state of a dataset, if it's published or not and the running operation. It is a finite state machine, with the following properties:
+
+* It is in a single state at a time
+* The list of possible states is fully known
+* The list of all transitions is fully known, each with the actions and conditions that can trigger them
+
+Here is the full state machine description. 
+
+![Dataset states machine](dataset_status_states.png "Dataset states machine")
+
+### Commons attributes
+
+Attribute | Description
+--------- | -----------
+`published` <br> *boolean*       | True if the dataset is available in the search API.
+`status` <br> *string*      | One of the dataset status values
+`since` <br> *datetime* | Timestamp writed the dataset entered in the current status
+`user` <br> *user* <br> <em class="expandable">expandable</em> | User who started the action
+
+## Retrieve the current dataset status
+
+```HTTP
+GET https://{YOURDOMAIN}.opendatasoft.com/api/management/v2/datasets/{DATASET_UID}/status
+```
+
+> Example request
+
+```HTTP
+curl https://yourdomain.opendatasoft.com/api/management/v2/datasets/da_XXXXXX/status \
+    -u username:password
+```
+
+> Example response
+
+```json
+{
+    "published": True,
+    "status": "processing",
+    "since": "2015-04-15T15:13:04+00:00"
+}
+```
+
+Retrieves the current dataset status.
+
+### Parameters
+
+No parameters
+
+### Returns
+
+The dataset status object that applies to the given dataset. See [Dataset status](#dataset-status) for specific attributes and the list of objects.
+
+## The `idle` dataset status
+
+> Example object
+
+```json
+{
+    "published": True,
+    "status": "idle",
+    "since": "2015-04-15T15:13:04+00:00"
+}
+```
+
+The *idle* status means that the dataset does not performed a specific action. It can be published or not but no job is running. A newly created dataset is in that status.
```

Comment:
-does not performed a specific action +is not currently involved in a processing ?